pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -220,9 +220,21 @@ func (c *ClusterMemberController) getEtcdPodToAddToMembership() (*corev1.Pod, er
 			continue
 		}
 
+		// check to see if this member is updating from 4.3
+		etcdMember, err := c.etcdClient.GetMember(pod.Spec.Hostname)
+		switch {
+		case apierrors.IsNotFound(err):
+			// this is not an upgrade from 4.3
+		case err != nil:
+			return nil, err
+		default:
+			klog.Infof("skipping unready pod %q because it is already an etcd member: %#v with hostname: %s", pod.Name, etcdMember, pod.Spec.Hostname)
```

Comment:
will this be constantly printed in the upgrade cases?  if so, V(4)

Reply:
I think it will only be temporary and very rare to find. Can still push it to V(4)